docs/modules/help/partials/faq_ticket_detail.adoc

```diff
@@ -1,3 +1,29 @@
+[[closing]]
 === Why can't I close some tickets?
 
 Some tickets might be in such a state that can't be closed. Only new tickets, tickets in "backlog" and in "customer review" status can be closed. If you need to close such a ticket, please write a comment requesting to change the state.
+
+[[secrets]]
+=== How can I add secrets (passwords, keys, etc.) to tickets?
+
+To communicate secrets to VSHNeers with a secure mechanism, follow these steps:
+
+. After creating a new ticket, click on it on the list of tickets, and scroll down until the section "Secrets" is visible.
+. Click on the btn:[Add Secret] button.
+. Enter a name for the secret in the **Name** field.
+. Enter the value of the secret in the **Secret** field.
```

Comment:
Think this can be further specified like
```
. Enter a human-friendly name for the secret in the **Name** field. 
. Enter the value of the secret in the **Secret** field.
  You can enter any free-text information here, e.g. SSH keys, environment variables, certificates, etc.
```